command.go

```diff
@@ -177,6 +180,17 @@ func (c *commandContext) SetPointSize(ps float32) error {
 	return nil
 }
 
+func (c *commandContext) NumFastRenderPoints() int {
+	return c.numFastRenderPoints
+}
+
+func (c *commandContext) SetNumFastRenderPoints(n int) error {
```

Comment:
Playing around a bit with this, I thought we could set a minimum value to prevent the pointcloud from completely disappearing.